### PR TITLE
Issue500 Dynamic balance

### PR DESCRIPTION
**Pull request by GlennReynders**
removed unknown references to dynamic balance from ventilation system and basecircuits

### CODE REVIEW

File: IDEAS/Templates/BaseCircuits/PumpSupply_dp.mo [hunk 1]
```diff
@@ -20,7 +20,6 @@ model PumpSupply_dp
       riseTime=riseTime,
       init=init,
       massDynamics=Modelica.Fluid.Types.Dynamics.SteadyState,
-      motorCooledByFluid=false,
```
Comment (damienpicard) [inline IDEAS/Templates/BaseCircuits/PumpSupply_dp.mo]: @GlennReynders is it on purpose that you want to remove the `motorCooledByFluid`?

**Comment by damienpicard** [comment]
@GlennReynders once the comment is answered, this can be merged.

File: IDEAS/Templates/BaseCircuits/PumpSupply_m_flow.mo [hunk 2]
```diff
@@ -18,8 +18,7 @@ model PumpSupply_m_flow "Pump on supply duct"
       flowRegulator(
         filteredSpeed=filteredSpeed,
         riseTime=riseTime,
-        init=init,
-        motorCooledByFluid=motorCooledByFluid));
```
Comment (damienpicard) [inline IDEAS/Templates/BaseCircuits/PumpSupply_m_flow.mo]: @GlennReynders idem

File: IDEAS/Templates/BaseCircuits/PumpSupply_dp.mo (hunk 1 above)
Comment (GlennReynders) [inline IDEAS/Templates/BaseCircuits/PumpSupply_dp.mo]: yes, because the parameter no longer exists

**Pull request merged**